src/python/espressomd/script_interface.pyx

```diff
@@ -127,12 +128,15 @@ cdef class PScriptInterface(object):
             return v
         elif type(value) == str:
             return make_variant[string](to_char_pointer(value))
-        elif type(value) == int:
-            return make_variant[int](value)
-        elif type(value) == float:
-            return make_variant[float](value)
+        
+        # Note: Bool has to be checked before int/float because
+        # numpy considers True to be an instance of np.int
         elif type(value) == type(True):
             return make_variant[bool](value)
+        elif type(value) == int or isinstance(value,np.int):
```

Comment:
Probably it should be
```python
elif np.issubdtype(np.dtype(type(value)), int):
```
This interactive session at least looks good (Output in Python 2 and 3 is the same).  It also solves the problem that `bool` is falsely recognized as `np.in`.
```python
>>> value = 2
>>> np.issubdtype(np.dtype(type(value)), int)
True
>>> value = np.int64(2)
>>> np.issubdtype(np.dtype(type(value)), int)
True
>>> value = 2.0
>>> np.issubdtype(np.dtype(type(value)), int)
False
>>> value = np.float64(2.0)
>>> np.issubdtype(np.dtype(type(value)), int)
False
>>> value = True
>>> np.issubdtype(np.dtype(type(value)), int)
False
>>> value = "Definitely not a number"
>>> np.issubdtype(np.dtype(type(value)), int)
False
```